src/plugins/manualRowResize/manualRowResize.js

```diff
@@ -292,14 +293,14 @@ class ManualRowResize extends BasePlugin {
    * @returns {boolean}
    */
   checkIfRowHeader(element) {
```

Comment:
As in the previous comment maybe this part of code could be replace to `return closest(element, ['TH', 'TBODY'], this.hot.rootElement) ? true : false;` WDYT?